src/profiles.js

```diff
@@ -0,0 +1,77 @@
+import tv4 from 'tv4'
+import jsonlint from 'json-lint'
+import _ from 'lodash'
+
+import Registry from './registry'
+
+class Profiles {
```

Comment:
So plan is to have merged`Profiles` and `Registry` (make `Regisry` methods as private `Profiles` methods) classes. It have the same responsibility and term registry is kinda confusing because there is datapackage registry also.